examples/llava/mtmd-cli.cpp

```diff
@@ -308,7 +308,7 @@ int main(int argc, char ** argv) {
             }
             g_is_generating = true;
             if (line.find("/image") == 0) {
-                std::string image = line.substr(7);
+                std::string image = (line.size()>=7) ? line.substr(7) : "";
```

Comment:
hmm this logic looks a bit hacky, I will replace it with something cleaner

Reply:
Sure, please. Thanks for checking out.